File: Currency-Converter/static/convert.js
//USING JAVASCRIPT AXIOS (ALTERNATIVE OPTION)

// resultSection = $('#coverted-result');

// $('form').on('submit', async function (evt) {
//   evt.preventDefault();
//   fromValue = $('#from-value').val();
//   toValue = $('#to-value').val();
//   inputValue = $('#input-value').val();
//   res = await convert(fromValue, toValue, inputValue);
//   $(resultSection).append(
//     `<h2>Your converted value is <input type="text" disabled value="${res} ${toValue}" /></h2>`
//   );
//   this.reset();
// });

// async function convert(from, to, value) {
//   const res = await axios({
//     method: 'GET',
//     url: 'https://api.exchangerate.host/convert',
//     params: {
//       from: from,
//       to: to,
//       amount: value,
//     },
//   });
//   return res.data.result;
// }

File: Currency-Converter/convert.py
from flask import Flask, render_template, jsonify, json, request, redirect
import requests

baseURL = 'https://api.exchangerate.host/'


def get_currency(dict):
    for key in dict:
        return [key for key in dict.keys()]


def convert(from_value, to_value, amount):
    res = requests.get(f'{baseURL}/convert', params={'from': from_value,
                                                     'to': to_value, 'amount': amount})
    return res
